gapis/gfxapi/vulkan/vulkan.api

```diff
@@ -9132,6 +9142,7 @@ enum RecordingState {
   @unused VkDescriptorPoolCreateFlags     Flags
   @unused u32                             MaxSets
   @unused map!(u32, VkDescriptorPoolSize) Sizes
+  @unused map!(VkDescriptorSet, ref!DescriptorSetObject) DescriptorSets
```

Comment:
Right, but this means we will need to traverse through all the descriptorsets in a pool when we want to free a specific descriptorset.